tasks/src/test/java/org/example/aProblemAboutEquation/AProblemAboutEquation3Test.java

```diff
@@ -0,0 +1,72 @@
+package org.example.aProblemAboutEquation;
+
+import lombok.extern.java.Log;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+@Log
+class AProblemAboutEquation3Test {
+    private final AProblemAboutEquation3 aProblemAboutEquation3 = new AProblemAboutEquation3();
+
+    @BeforeAll
+    static void showMessage() {
+        log.info("This is my first test's class.");
+    }
+
+    @Test
+    void shouldReturnFalseWhenNumberOfFriendsLessThanTwoOrGreaterThan100() {
+        //given - arrange
+        int numberOfFriends = 1;
+
+        //when - act
+        boolean numberOfFriendsVerifier = aProblemAboutEquation3.numberOfFriendsVerifier(numberOfFriends);
+
+        //then - assert
+        Assertions.assertFalse(numberOfFriendsVerifier);
+    }
+
+    @Test
+    void shouldReturnFalseWhenNumberOfMillilitersOfTheDrinkInTheBottleLessThanOneOrGreaterThan100() {
+        //given - arrange
+        int millilitersOfTheDrinkInTheBottle = 101;
+
+        //when - act
+        boolean millilitersOfTheDrinkInTheBottleVerifier = aProblemAboutEquation3.millilitersOfTheDrinkInTheBottleVerifier(millilitersOfTheDrinkInTheBottle);
```

Comment:
sformatuj to jakos bo zle to wyglada. Ze pod spodem od lewej od krawedzi jest kod od razu

Reply:
U mnie w IntelliJ-u jest OK.